vs-17-2022-win64-cxx17.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2015-2022, Ruslan Baratov, Rahul Sheth
+# All rights reserved.
```

Comment:
Are these comments valid here?

Reply:
They are the original that we copied from.